crates/selene-lang/src/ty.rs

```diff
@@ -17,6 +18,7 @@ impl fmt::Display for Type {
             Type::Int => write!(f, "Int"),
             Type::Float => write!(f, "Float"),
             Type::Bool => write!(f, "Bool"),
+            Type::List(_) => write!(f, "List"),
```

Comment:
This can probably be `[{ty}]` or however we decide to represent it later, but we can figure that out when we come up with a list syntax.